### ISSUE TITLE
kubeblocks.io site missing KubeBlocks' operator's monitoring howto

### ISSUE THREAD

**Issue by nashtsai** [apecloud/kubeblocks.io#136]
kubeblocks.io site missing KubeBlocks' operator's monitoring howto.

ref: https://kubedb.com/docs/v2023.08.18/setup/monitoring/overview/

**Comment by michelle-0808** [comment]
@nashtsai Does this doc meet your needs?
https://kubeblocks.io/docs/preview/user_docs/observability/monitor-database

**Comment by nashtsai** [comment]
> @nashtsai Does this doc meet your needs? https://kubeblocks.io/docs/preview/user_docs/observability/monitor-database

nope, do check https://sdk.operatorframework.io/docs/best-practices/observability-best-practices/ ,  and Liqo's user guides on for Monitoring (https://docs.liqo.io/en/v0.10.1/usage/prometheus-metrics.html)

**Comment by michelle-0808** [comment]
@nashtsai Got it. I also reported this issue to our product manager. We plan to add the monitoring KubeBlocks operator function in 0.9.x or later and docs also will be added then.
